src/components/include/sl.h

```diff
@@ -462,15 +462,16 @@ sl_cs_exit_schedule_nospin_arg(struct sl_thd *to)
 		}
 	}
 
-	assert(t->state == SL_THD_RUNNABLE);
+	//WOKE is subset of RUNNABLE so should be safe to dispatch to
```

Comment:
`/* */` is the prescribed comment style for single-line comments.

Reply:
I don't think that describing the WOKE state as a *subset* is correct.  Comment *could* read:

`/* WOKE threads are still runnable */`

Likely the extensible way to handle this is to have a function `sl_is_runnable(t)` that can encode which states include threads that can be dispatched to.